airbyte-integrations/connectors/source-mongodb-v2/metadata.yaml

```diff
@@ -22,6 +22,9 @@ data:
   tags:
     - language:java
   releases:
+    1.0.1:
+      message: >
+        Patched bug where passwords with special characters were not being handled correctly.
```

Comment:
This should go to changelog in [mongodb-v2.md](https://github.com/airbytehq/airbyte/blob/master/docs/integrations/sources/mongodb-v2.md?plain=1)